bgpd/bgp_evpn_vty.c

```diff
@@ -5950,6 +5943,16 @@ DEFUN (bgp_evpn_vrf_rt,
 		return CMD_WARNING_CONFIG_FAILED;
 	}
 
+	if (rt_type == RT_TYPE_BOTH || rt_type == RT_TYPE_EXPORT) {
```

Comment:
nit: Why not just check against RT_TYPE_IMPORT?

Reply:
Done, thanks.